tests/helpers.py

```diff
@@ -609,7 +609,28 @@ def get_all_pvs():
     return ocp_pv_obj.get()
 
 
-@retry(AssertionError, tries=10, delay=5, backoff=1)
+def delete_pvcs(pvc_objs, wait=True):
+    """
+    Deletes list of the pvc objects
+
+    Args:
+        pvc_objs (list): List of the pvc objects to be deleted
+        wait (bool): True if need to wait for PVCs deletion
+
+    Returns:
+        bool: True if deletion is successful
+    """
+    status = pvc.delete_pvcs(pvc_objs)
+    if wait:
+        for pvc_obj in pvc_objs:
+            if pvc_obj.backed_pv_obj().get('spec').get(
```

Comment:
This validation always fail since backed_pv_obj method will reloads the pvc instance which is already deleted in line 623